src/components/VrxGrid/VrxGrid.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div data-testid="vrx-grid" class="relative overflow-x-auto h-full shadow-md w-full rounded-lg bg-white dark:bg-gray-800">
+    <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
+      <VrxGridHeader v-model:grid-config="gridModel.configuration"/>
+      <VrxGridBody v-model="gridModel.configuration"/>
+    </table>
+  </div>
+</template>
+
+<script setup lang="ts">
+  import VrxGridHeader from "@/components/VrxGrid/VrxGridHeader.vue";
+  import VrxGridBody from "@/components/VrxGrid/VrxGridBody.vue";
+  import {GridConfiguration, GridRow} from "@/components/VrxGrid/GridConfiguration.ts";
+  import {onMounted, provide} from "vue";
+  import {Grid} from "@/components/VrxGrid/Models/Grid.ts";
+
+  onMounted(() => {
+    document.addEventListener('keydown', (e) => {
```

Comment:
hai provato ad aggiungere questo listenere al div di vrx-grid e non a tutta la pagina? in questo modo dovrebbe funzionare solo quando hai il focus sulla grid e non sempre